.github/SUPPORT.md

```diff
@@ -0,0 +1,9 @@
+# PowerShell Support  
+If you have any problems, please consult the [known issues][], developer [FAQ][], and [GitHub issues][].  
+If you do not see your problem captured, please file a [new issue][] and follow the provided template.  
+
+[FAQ]: https://github.com/PowerShell/PowerShell/tree/master/docs/FAQ.md
```

Comment:
Add a to the PowerShell Core life cycle which include the official support policy:
https://docs.microsoft.com/en-us/powershell/scripting/powershell-support-lifecycle?view=powershell-6

Reply:
👍